code/modules/reagents/holder.dm

```diff
@@ -504,9 +504,11 @@
 /datum/reagents/proc/reaction(atom/A, method=TOUCH, volume_modifier=1, show_message = 1)
 	var/react_type
 	if(isliving(A))
+		var/mob/living/L = A
+		if (L.stat == DEAD)
+			return
 		react_type = "LIVING"
 		if(method == INGEST)
-			var/mob/living/L = A
 			L.taste(src)
```

Comment:
why?

Reply:
The check itself is needed to avoid corpses soaking up reagents (which was being used to dupe chems).
You would think the (isliving) check would prevent that but it doesn't, hence the L.stat == DEAD check.